build-output-api/edge-functions/README.md

```diff
@@ -2,6 +2,8 @@
 
 ### Build Output API
 
+#### [Demo](https://build-output-api-edge-functions.vercel.sh)
```

Comment:
Should these be deployed to the examples team instead?

Reply:
@styfle examples in this repo live in the Vercel org, they started in vercel-solutions but we moved them months ago.